src/output.cpp

```diff
@@ -14,7 +15,23 @@ void clearScreen() {
     [[maybe_unused]] const auto test = system("clear");
 #endif
 }
+/*
+    Used to count the ships on both boards and displays them on the screen
+*/
+void healthBar(const BoardType& playerBoard, const BoardType& computerBoard) {
+    size_t playerHealth = 0, computerHealth = 0;
+    for (const auto& segment : playerBoard)
+        playerHealth +=
+            std::count(segment.begin(), segment.end(), FieldValue::SHIP);
+
+    for (const auto& segment : computerBoard)
+        computerHealth +=
+            std::count(segment.begin(), segment.end(), FieldValue::SHIP);
```

Comment:
Please extract this into it's own function. This kind of repeated code should be avoided.